app/models/energy_tariff.rb

```diff
@@ -89,6 +91,10 @@ def value_for_charge(type)
 
   private
 
+  def tariff_holder_symbol
+    meters.any? ? :meter : tariff_holder_type.underscore.to_sym
```

Comment:
something similar to this would be useful in the editor bits i'm working on so i'm wondering if maybe this could be a public method?

e.g. I'm doing things like this in a few places
```
      case @energy_tariff.tariff_holder_type
      when 'School' then redirect_to school_energy_tariff_energy_tariff_flat_prices_path(@school, @energy_tariff)
      when 'SiteSettings' then redirect_to admin_settings_energy_tariff_energy_tariff_flat_prices_path(@energy_tariff)
      end
```